config/mail.php

```diff
@@ -1,123 +1,108 @@
-
-    'encryption' => env('MAIL_ENCRYPTION', 'tls'),
-
-    /*
-    |--------------------------------------------------------------------------
-    | SMTP Server Username
-    |--------------------------------------------------------------------------
-    |
-    | If your SMTP server requires a username for authentication, you should
-    | set it here. This will get used to authenticate with your server on
-    | connection. You may also set the "password" value below this one.
-    |
-    */
-
-    'username' => env('MAIL_USERNAME'),
-
-    'password' => env('MAIL_PASSWORD'),
-
-    /*
-    |--------------------------------------------------------------------------
-    | Sendmail System Path
-    |--------------------------------------------------------------------------
-    |
-    | When using the "sendmail" driver to send e-mails, we will need to know
-    | the path to where Sendmail lives on this server. A default path has
-    | been provided here, which will work well on most of your systems.
-    |
-    */
-
-    'sendmail' => '/usr/sbin/sendmail -bs',
-
-    /*
-    |--------------------------------------------------------------------------
-    | Markdown Mail Settings
-    |--------------------------------------------------------------------------
-    |
-    | If you are using Markdown based email rendering, you may configure your
-    | theme and component paths here, allowing you to customize the design
-    | of the emails. Or, you may simply stick with the Laravel defaults!
-    |
-    */
-
-    'markdown' => [
-        'theme' => 'default',
-
-        'paths' => [
-            resource_path('views/vendor/mail'),
-        ],
-    ],
-
+	/*
+	|--------------------------------------------------------------------------
+	| Default Mailer
+	|--------------------------------------------------------------------------
+	|
+	| This option controls the default mailer that is used to send any email
+	| messages sent by your application. Alternative mailers may be setup
+	| and used as needed; however, this mailer will be used by default.
+	|
+	*/
+
+	'default' => env('MAIL_MAILER', 'smtp'),
+
+	/*
+	|--------------------------------------------------------------------------
+	| Mailer Configurations
+	|--------------------------------------------------------------------------
+	|
+	| Here you may configure all of the mailers used by your application plus
+	| their respective settings. Several examples have been configured for
+	| you and you are free to add your own as your application requires.
+	|
+	| Laravel supports a variety of mail "transport" drivers to be used while
+	| sending an e-mail. You will specify which one you are using for your
+	| mailers below. You are free to add additional mailers as required.
+	|
+	| Supported: "smtp", "sendmail", "mailgun", "ses",
+	|            "postmark", "log", "array"
+	|
+	*/
+
+	'mailers' => [
+		'smtp' => [
+			'transport' => 'smtp',
+			'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
```

Comment:
We don't care because we never use this, correct?

Reply:
exactly.